test/unit/model/test_model_creator.py

```diff
@@ -35,3 +36,27 @@ def test_create_model_unknown_model(self):
 
         with self.assertRaises(ValueError):
             ModelCreator.create_model(model_type)
+
+
+class TestMLModelInterfaces(TestCase):
+    """Test if all ml models defined in openstf.model.ml_model
```

Comment:
Deze unit test check op attributen is (als het goed is) overbodig als de modellen afhankelijk zijn van de AbstractMLModel. Enkel het initieren van de klasse doet dan ook deze checks al.